### PR TITLE
fix: updated ui section of environment list[123]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description
1.In this PR I have refactored the UI of the environment list from figma also fixes the bug of overlapping of Environment name dropdown with body of the variables
2.I also fixes a UI bug in the collection where Request type and Request is too close to each other

Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@library/ui/options/Options.svelte [hunk 1]
```diff
@@ -56,10 +56,10 @@
   }
 </script>
 
-<nav style="position: fixed; top:{mouseY}px; left:{mouseX}px; z-index:4;">
+<nav style="position: fixed; top:{mouseY}px; left:{mouseX}px; z-index:500;">
   <div
     style={`width: ${noOfColumns}px`}
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/options/Options.svelte:61]: take zIndex from props

File: src/packages/@library/ui/options/Options.svelte [hunk 2]
```diff
@@ -56,10 +56,10 @@
...
-    class="overflow-hidden navbar pb-0 pt-0 d-flex flex-column border-radius-2 align-items-start justify-content-start text-whiteColor bg-tertiary-400"
+    class="overflow-hidden navbar pb-0 pt-0 d-flex flex-column border-radius-2 align-items-start justify-content-start text-whiteColor bg-tertiary-400 z-in"
   >
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/options/Options.svelte:63]: z-in ??

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**